redash/destinations/microsoft_teams_webhook.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import requests
+
+from redash.destinations import *
+from redash.utils import json_dumps
+from redash.serializers import serialize_alert
+
+
+class MicrosoftTeamsWebhook(BaseDestination):
+    @classmethod
+    def name(cls):
+        return "Microsoft Teams Webhook"
+
+    @classmethod
+    def type(cls):
+        return "microsoft_teams_webhook"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {"type": "string"},
```

Comment:
You might want to add a `title` here to indicate this is a webhook URL.

Reply:
Done.

<img width="517" alt="Screen Shot 2022-01-26 at 3 23 28 PM" src="https://user-images.githubusercontent.com/3395738/151263911-0c783015-3695-42cb-aa31-e696eb66a44c.png">